### DISCUSSION TITLE
分页组件可选的展示方法

### DISCUSSION THREAD

**Discussion by zms9110750** [BlazorComponent/MASA.Blazor#1244] [Ideas]
- 将..集成到首末按钮中，而不是作为空白显示。`1..` 和 `..100`。
- 外置` |< `和` >| `按钮，分在上下页按钮的更外侧。表示首末页。按钮上的`..`就不需要了。
- 只显示端页，而不连同它周围的页面都显示。例如显示`1,2,3,4,5, ..100`而不是` 1,2,3, .. 98,99,100 `。在12的时候都显示13，14，15.但是到了3，4页却只显示15页了。这个过度太奇怪了。

**Comment by doddgu** [comment]
我们目前只对vuetify进行了功能上的还原，但对国内开发者来说或许不是大家习惯的方式。我们欢迎pr，对于功能扩展我们是开放的。
